### PR TITLE
Add `--no-std` option

### LINKED ISSUE #234: Add --no-std flag
Add a flag to prevent including the standard library

### PR DESCRIPTION
**Pull request by Yumasi**
Closes #234.

This adds the `-no-std` flag to jinko. In order to do this, the context creation had to be moved out of the parsing function.

### CODE REVIEW

File: src/context.rs [hunk 1]
```diff
@@ -82,18 +82,7 @@ impl Context {
...
-        let mut stdlib_incl =
-            crate::instruction::Incl::new(String::from("stdlib"), Some(String::from("")));
-        stdlib_incl.set_base(PathBuf::new());
-        // FIXME: Remove unwrap
-        ctx.entry_point
-            .add_instruction(Box::new(stdlib_incl))
-            .unwrap();
-
-        ctx
+        Context::empty()
```
Comment (CohenArthur) [inline src/context.rs:85]: I think we should remove entirely one of the functions, either `new` or `empty`. This can be done in this PR or in another one

**Review by CohenArthur — APPROVED** [review]
Thanks a lot! LGTM besides one nitpick

File: src/context.rs (hunk 1 above)
Comment (Yumasi) [reply src/context.rs:85]: I agree. I would keep `new` (seems more idiomatic I think). I will update this PR as soon as possible.

**Review by Skallwar — APPROVED** [review]

**Review by CohenArthur — APPROVED** [review]
Thanks a lot!!!

**Pull request merged**